src/app/shared/comment-editor/comment-editor.component.ts

```diff
@@ -18,14 +18,23 @@ export class CommentEditorComponent implements OnInit {
 
   @Input() commentField: AbstractControl;
   @Input() commentForm: FormGroup;
+  @Input() id: string;
+  @Input() initialDescription?: string;
   @ViewChild('dropArea') dropArea;
   @ViewChild('commentTextArea') commentTextArea;
   @ViewChild('markdownArea') markdownArea;
 
   dragActiveCounter = 0;
   uploadErrorMessage: string;
 
-  ngOnInit() {}
+  ngOnInit() {
+    if (this.initialDescription !== undefined) {
+      this.commentField.setValue(this.initialDescription);
+    }
+    if (this.id === undefined) {
+      throw new Error('Comment Editor\'s FormName ([id]) has not been specified');
```

Comment:
Should we do the same thing for the other compulsory `@Input` in CommentEditor? :)

Reply:
Sure! Done.